packages/react-native-sdk/docusaurus/docs/reactnative/02-guides/12-running-calls-in-background.mdx

```diff
@@ -3,4 +3,170 @@ id: 12-running-calls-in-background
 title: Running Calls in Background
 ---
 
-### Stay tuned for more information on this topic.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ImageShowcase from '@site/src/components/ImageShowcase';
+import iosInfoPlist from '../assets/ios-info-plist.png'
+
+A natural expected behavior during a call is to keep being on the call even when the app goes to the background. In this guide, we explain how to achieve this in iOS and Android.
+
+<Tabs
+  defaultValue='android'
+  values={[
+    { label: 'Android', value: 'android' },
+    { label: 'iOS', value: 'ios' },
+  ]}
+>
+<TabItem value='android'>
+
+In Android, when the app goes to the background, we can run a foreground service to keep the call alive. This ensures that both the video stream and audio stream are kept alive while the app goes into the background.
+
+We can start a foreground service using the [notifee library](https://notifee.app/).
+
+#### Install notifee
+
+<Tabs groupId="package-manager">
+  <TabItem value="Yarn" default>
+
+```bash
+  yarn add @notifee/react-native
+  npx pod-install
+```
+
+  </TabItem>
+  <TabItem value="npm">
+
+```bash
+  npm install @notifee/react-native
+  npx pod-install
+```
+  </TabItem>
+    </Tabs>
+
+#### Configure the foreground service
+
+Add the following entry to your `AndroidManifest.xml`
+
+```xml
+    <service
+      android:name="app.notifee.core.ForegroundService"
+      android:foregroundServiceType="camera|microphone" />
+```
+
+Add the following function in your codebase to start a foreground service and to keep the call alive.
+
```

Comment:
I find that lib. Specific installation steps should not be mentioned in our docs, yet we should send our integrators via a link to the relevant section in Notiffee's docs.

Otherwise, it's just to difficult to maintain.

Reply:
this is specific to our usecase unfortunately 

```
android:foregroundServiceType="camera|microphone"
```